engines/fullpipe/scenes/scene18and19.cpp

```diff
@@ -22,17 +22,16 @@
 
 #include "fullpipe/fullpipe.h"
 
-#include "fullpipe/objectnames.h"
+#include "fullpipe/behavior.h"
 #include "fullpipe/constants.h"
-
 #include "fullpipe/gameloader.h"
+#include "fullpipe/interaction.h"
 #include "fullpipe/motion.h"
+#include "fullpipe/objectnames.h"
 #include "fullpipe/scenes.h"
 #include "fullpipe/statics.h"
 
-#include "fullpipe/interaction.h"
-#include "fullpipe/behavior.h"
-
```

Comment:
I am not sue about reordering the includes alphabetically here. There is sometimes a logic behind the order in which they are done.